### ISSUE TITLE
Chakra Netlify Storybook Version Should Be Updated

### ISSUE THREAD

**Issue by ZiedHf** [chakra-ui/chakra-ui#3925]
## 🧱 Problem Statement / Justification
I'm using the [Storybook Composition](https://storybook.js.org/docs/react/workflows/storybook-composition) in my Storybook App.

Due to a CORS problem [Chakra Storybook](https://chakra-ui.netlify.app) returns an error whenever I run my Storybook App with composition.

![image](https://user-images.githubusercontent.com/7974028/116308775-e92d3b80-a775-11eb-973c-9cd073859bcf.png)

I guess this issue is related to [#11892 on Storybook repo](https://github.com/storybookjs/storybook/issues/11892#issuecomment-674409716)

## ✅ Proposed solution or API

As described above, it's known issue related to Storybook v5. So I guess an update from the version 5.2.4 to the latest version will resolve the problem.
In the package.json file sb@6.2 is used. Maybe we need only to deploy it and update the version of [Chakra Netlify](https://chakra-ui.netlify.app/?path=/settings/about) ?
![image](https://user-images.githubusercontent.com/7974028/116309052-475a1e80-a776-11eb-9a70-c7cf7bfded54.png)

## 📝 Additional Information

Current error
![image](https://user-images.githubusercontent.com/7974028/116309288-9738e580-a776-11eb-9c5c-fb766316d922.png)

**Comment by lakshmimani** [comment]
![Screenshot 2021-05-21 at 3 33 32 PM](https://user-images.githubusercontent.com/13865074/119120453-ea3a4b80-ba49-11eb-9916-91d0cddcc45f.png)
Upgraded to 6.2.9 still the issue persist.

**Comment by ZiedHf** [comment]
@lakshmimani This [one](https://chakra-ui.netlify.app/?path=/settings/about) should be upgraded not only yours.

**Comment by ervwalter** [comment]
This is still an issue for me

**Comment by mtsmachado8** [comment]
It's still an issue for me too. please reopen it

**Comment by WarKaa** [comment]
Still an issue for me too

**Comment by TimKolberger** [comment]
I just configured a CI deployment of our latest stories on the main branch to https://storybook.chakra-ui.com/

**Comment by mtsmachado8** [comment]
Nice work!!! it looks much better now @TimKolberger! Chackra-ui stories are updated now right?
![image](https://user-images.githubusercontent.com/11022437/129460026-190f43bf-cd3f-49ed-8528-cdac3d5924b4.png)
I noticed it doesn't have controls enabled for your stories... iam making a Design System based on Materia-Ui and Storybook for my company. I would really enjoy helping you with some storybook stuff.
I can help changing the storybook colors and logo on this deploy so it can look more like Chackra-Ui Branding on this deploy: https://storybook.chakra-ui.com/
I did something similar to my company's storybook:
![image](https://user-images.githubusercontent.com/11022437/129460055-10d409e3-2c72-43aa-b05d-3205aaff7a64.png)
